arm_id: $(arg arm_id)

m_ee: 0.76
franka_gripper:
  type: franka_gazebo/FrankaGripperSim
  arm_id: $(arg arm_id)

  finger1:
    gains: { p: 100, i: 25, d: 20 }

  finger2:
    gains: { p: 100, i: 25, d: 20 }
